app/src/main/java/ca/fuwafuwa/kaku/Ocr/OcrRunnable.java

```diff
@@ -137,26 +140,11 @@ public void stop(){
         }
     }
 
-    private Bitmap getReadyScreenshotBox(BoxParams box, int attempts) throws OutOfMemoryError, StackOverflowError, TimeoutException, FileNotFoundException {
-
-        if (attempts > 10){
-            return null;
-        }
+    private Bitmap getReadyScreenshotBox(BoxParams box) throws OutOfMemoryError, StackOverflowError, TimeoutException, FileNotFoundException {
```

Comment:
There's also a couple "gotcha's" here as well.

For example, `mContext.getScreenshot()` sometimes will return NULL if the display buffer isn't ready to give you a frame (which will happen when you continuously request screenshots to check for where the capture box location is), so that needs to be handled accordingly as well.